features/language-core.feature

```diff
@@ -214,7 +214,8 @@ Feature: Manage translation files for a WordPress install
       """
     And the return code should be 0
 
-  @require-php-5.6 @less-than-php-7.0
+  # This test downgrades to older WP version. PHP restriction is added to avoid warnings in latest versions.
+  @require-wp-4.0 @less-than-php-7.0
```

Comment:
Is the `@require-wp-4.0` restriction really needed?

The test upgrades to 4.0 anyway.

This way we don't need this `load-wp-cli.feature` file...

Reply:
Removed.